tests/config/test_configtypes.py

```diff
@@ -328,38 +333,39 @@ class BoolTests(unittest.TestCase):
 
     INVALID = ['10', 'yess', 'false_']
 
-    def setUp(self):
+    @pytest.fixture(autouse=True)
+    def setup(self):
         self.t = configtypes.Bool()
 
-    def test_transform(self):
+    _ALL_FORMS = []
```

Comment:
Should we have a method for this and call it in ``pytest.mark.parametrize`` :question: